src/cli/commands/lint.ts

```diff
@@ -109,28 +107,50 @@ linting ./openapi.yaml
   }
 }
 
+async function handleReaderError(command: Lint, reader: Function) {
```

Comment:
I think this is not a great name; it's not handling the error, but it's rather trying to read the value and eventually handle the error.

```suggestion
async function tryReadOrLog(command: Lint, reader: Function) {
```
or something like that